### PR TITLE
Fix scripts.

### PR DESCRIPTION
**Pull request by smillst**

### CODE REVIEW

File: annotation-file-utilities/scripts/insert-annotations [hunk 1]
```diff
@@ -28,8 +28,8 @@ while [ "$#" -gt 0 ]; do
 done
 
 AFU=${AFU:-$(dirname $0)/..}
-ANNOTATION_FILE_UTILS=${AFU}/bin:${AFU}/../scene-lib/bin:${AFU}/../asmx/bin:${AFU}/annotation-file-utilities-all.jar
-JAVAC_JAR=${JAVAC_JAR:-${ANNOTATION_FILE_UTILS}/lib/compiler-2.4.0.jar}
+ANNOTATION_FILE_UTILS=${AFU}/annotation-file-utilities-all.jar
+JAVAC_JAR=${JAVAC_JAR:-${AFU}/lib/compiler-2.4.0.jar}
```
Comment (smillst) [inline annotation-file-utilities/scripts/insert-annotations]: JAVAC_JAR was set to `${AFU}/bin:${AFU}/../scene-lib/bin:${AFU}/../asmx/bin:${AFU}/annotation-file-utilities-all.jar/lib/compiler-2.4.0.jar`.

File: annotation-file-utilities/scripts/insert-annotations [hunk 2]
```diff
@@ -38,9 +38,9 @@ if [ "$DEBUG" = "1" ]; then
...
   echo "JAVAC_JAR=${JAVAC_JAR}"
   # Keep this in sync with the actual command below.
-  echo "java -ea -cp ${JAVAC_JAR}:${AFU}/lib/plume-lib.jar:${ANNOTATION_FILE_UTILS}:${CLASSPATH} scenelib.annotations.io.classfile.ClassFileWriter $@"
+  echo "java -ea -cp ${ANNOTATION_FILE_UTILS}:${CLASSPATH} scenelib.annotations.io.classfile.ClassFileWriter $@"
   echo "--- end of insert-annotations debugging output"
 fi
 
 # Needs CLASSPATH to find user files
-java -ea -cp ${JAVAC_JAR}:${AFU}/lib/plume-lib.jar:${ANNOTATION_FILE_UTILS}:${CLASSPATH} scenelib.annotations.io.classfile.ClassFileWriter "$@"
+java -ea -cp ${ANNOTATION_FILE_UTILS}:${CLASSPATH} scenelib.annotations.io.classfile.ClassFileWriter "$@"
```
Comment (smillst) [inline annotation-file-utilities/scripts/insert-annotations]: JAVAC_JAR and lib/plume-lib.jar shouldn't be on the classpath.

File: annotation-file-utilities/scripts/insert-annotations-to-source [hunk 3]
```diff
@@ -29,8 +29,8 @@ done
 
 AFU=${AFU:-$(dirname $0)/..}
 AFU=`cd \`dirname $0\`/.. && pwd`
-ANNOTATION_FILE_UTILS=${ANNOTATION_FILE_UTILS:-${AFU}/bin:${AFU}/../scene-lib/bin:${AFU}/../asmx/bin:${AFU}/annotation-file-utilities-all.jar}
-JAVAC_JAR=${JAVAC_JAR:-${ANNOTATION_FILE_UTILS}/lib/compiler-2.4.0.jar}
+ANNOTATION_FILE_UTILS=${ANNOTATION_FILE_UTILS:-${AFU}/annotation-file-utilities-all.jar}
+JAVAC_JAR=${JAVAC_JAR:-${AFU}/lib/compiler-2.4.0.jar}
```
Comment (smillst) [inline annotation-file-utilities/scripts/insert-annotations-to-source]: JAVAC_JAR was set to ${AFU}/bin:${AFU}/../scene-lib/bin:${AFU}/../asmx/bin:${AFU}/annotation-file-utilities-all.jar/lib/compiler-2.4.0.jar.

**Comment by smillst** [comment]
Now that the build uses Gradle, `annotation-file-utilities/bin` and `scene-lib/bin` directories are not where classes are output.  (I guess they could be if that's how you configure your IDE.)  I added a comment explaining what every other change is fixing.

**Comment by wmdietl** [comment]
Thanks for the extra explanations and for finding these configuration issues!

Aren't some of these an oversight of #236 ? That is, the CF scripts, e.g. at:
https://github.com/typetools/checker-framework/blob/master/checker/bin-devel/javac#L29
point to the gradle output directories.
Also, the Eclipse `.classpath` file from the CF point to those directories:
https://github.com/typetools/checker-framework/blob/master/checker/.classpath#L9
whereas the AFU `.classpath` points to the now meaningless `bin` directory:
https://github.com/typetools/annotation-tools/blob/master/annotation-file-utilities/.classpath#L14
So shouldn't the AFU scripts and `.classpath` files point to the gradle output folder?

At the moment we have Eclipse IDE instructions at https://rawgit.com/typetools/checker-framework/master/docs/developer/developer-manual.html#Configure_Eclipse_to_edit_the_Checker_Framework
and it would be good to make sure they continue to work.

**Comment by smillst** [comment]
>Aren't some of these an oversight of #236 ? That is, the CF scripts, e.g. at:
https://github.com/typetools/checker-framework/blob/master/checker/bin-devel/javac#L29
point to the gradle output directories.

Yes. Though it only takes a second longer to compile the full jar, so I don't really see the need to have the output directories on the classpath.

As far as the Eclipse problems, I don't use Eclipse, so I'm not the best person to update those files.

**Pull request closed without merging**